.pre-commit-config.yaml

```diff
@@ -92,3 +92,9 @@ repos:
       ^[^/]+[.]rst$
     exclude: >-
       ^CHANGES\.rst$
+- repo: https://github.com/codespell-project/codespell
```

Comment:
As a pre-commit hook, does this run with the `-w` option to write changes? Or do we need to add an option? Pre-commit changes are automatically committed in CI, so the user doesn't need to do anything.

Reply:
great question/point -- I forgot that pre-commit can also change things (e.g. like black does) not just check. 

This one AFAIK is just a check.  I don't know (never looked) into how actually pre-commit "works" here but I guess it is via looking at https://github.com/codespell-project/codespell/blob/master/.pre-commit-hooks.yaml and there `entry: codespell` so it is indeed without `-w` if that is what it runs ;)